nixos/modules/services/network-filesystems/openafs/server.nix

```
@@ -303,9 +324,18 @@ in {
           mkdir -m 0755 -p /var/openafs
           ${optionalString (netInfo != null) "cp ${netInfo} /var/openafs/netInfo"}
           ${optionalString useBuCellServDB "cp ${buCellServDB}"}
+        '' + lib.optionalString (cfg.privilegedAdministrators != null) ''
+          rm -f /etc/openafs/server/UserList
         '';
         serviceConfig = {
           ExecStart = "${openafsBin}/bin/bosserver -nofork";
+          ExecStartPost = mkIf (cfg.privilegedAdministrators != null)
+            (pkgs.writeShellScript "openafs-admin-init" ''
+               for admin in ${lib.concatStringsSep " " cfg.privilegedAdministrators}
```

```suggestion
               for admin in ${lib.escapeShellArgs cfg.privilegedAdministrators}
```

Simpler and more correct.